digestset/deprecated.go

```diff
@@ -0,0 +1,51 @@
+package digestset
+
+import (
+	"github.com/opencontainers/go-digest"
+	"github.com/opencontainers/go-digest/digestset"
+)
+
+// ErrDigestNotFound is used when a matching digest
+// could not be found in a set.
+//
+// Deprecated: use [github.com/opencontainers/go-digest/digestset.ErrDigestNotFound].
+var ErrDigestNotFound = digestset.ErrDigestNotFound
+
+// ErrDigestAmbiguous is used when multiple digests
+// are found in a set. None of the matching digests
+// should be considered valid matches.
+//
+// Deprecated: use [github.com/opencontainers/go-digest/digestset.ErrDigestAmbiguous].
+var ErrDigestAmbiguous = digestset.ErrDigestAmbiguous
+
+// Set is used to hold a unique set of digests which
+// may be easily referenced by easily  referenced by a string
```

Comment:
```suggestion
// may be easily referenced by a string
```

Reply:
Do you want to address this one @thaJeztah? This conversation was marked resolved without changing the code comment.

Reply:
oh! overlooked it; mistook it for changing the documentation links; fixing now

Reply:
updated; now it's done _for real_ 😂